uaclient/clouds/aws.py

```diff
@@ -1,11 +1,14 @@
+import logging
 from typing import Any, Dict
 from urllib.error import HTTPError
 
 from uaclient import util
 from uaclient.clouds import AutoAttachCloudInstance
 
-IMDS_V2_TOKEN_URL = "http://169.254.169.254/latest/api/token"
-IMDS_URL = "http://169.254.169.254/latest/dynamic/instance-identity/pkcs7"
+IMDS_IP_ADDRESS = {"ipv4": "169.254.169.254", "ipv6": "[fd00:ec2::254]"}
```

Comment:
I don't think we need to make this a dict as we don't need to use the keys anywhere.
```suggestion
IMDS_IP_ADDRESS = ("169.254.169.254", "[fd00:ec2::254]")
```